plugins/techdocs/src/reader/components/TechDocsReaderPageHeader/TechDocsReaderPageHeader.tsx

```diff
@@ -102,7 +103,7 @@ export const TechDocsReaderPageHeader = (
   const labels = (
     <>
       <HeaderLabel
-        label="Component"
+        label="Entity"
```

Comment:
Could we use the Kind name (Component, Resource, API, etc.) instead and fall back to "Entity" if that's not provided? I'm sure this information is already available or easy to get at

Reply:
I found it and applied your suggestion!

Reply:
Awesome, thanks!